wiki/Installation/en.md

```diff
@@ -20,6 +22,19 @@ This page will roughly tell you how to get osu! working on your Windows device.
 5.  osu! will prompt you to sign in or register.
     -   If you have an account already, sign in.
     -   If you need to register, see [Registration](/wiki/Registration), then sign in.
+    
+### MacOS
+
+- Since the MacOS version of osu! is unofficial, the game may not fuction properly, or even boot at all. 
+
+1.  Go to the unofficial [download page](https://osu.ppy.sh/community/forums/topics/682197) and download the installer.
+2.  Locate the installer, right click, and select open.
+    - Double left clicking on the file may not open it, since MacOS may recognize the app as developed by an unidentified devloper. Right clicking and selecting open will solve this problem.
+3.  The installer will automatically run. You cannot change the installation location, since it is installed directly to `osu!.app`. Moving `osu.app` will move all of the files the come with it.
```

Comment:
```suggestion
3.  The installer will automatically run. The installation location cannot be changed. It will be installed directly to `osu!.app`. Moving `osu.app` will move all of the files the come with it.
```